test/readme.md

```diff
@@ -357,6 +361,23 @@ The following steps will walk you through how to run the tests for CSFLE.
       }
    }
    ```
+1. Start the KMIP servers:
+
+  `DRIVERS_TOOLS="/path/to/mongodb-labs/drivers-evergreen-tools" .evergreen/run-kms-servers.sh`
+
+1. Ensure default ~/.aws/config is present:
+
+  ```
+  aws_access_key_id = AWS_ACCESS_KEY_ID
+  aws_secret_access_key = AWS_SECRET_ACCESS_KEY
```

Comment:
```suggestion
  [default]
  aws_access_key_id=<AWS_ACCESS_KEY_ID>
  aws_secret_access_key=<AWS_SECRET_ACCESS_KEY.
```

I ran into two issues when following these instructions (probably because I blindly followed them without actually looking into the aws config documenation)

1. the default section of the config needs a header like `[default]`
2. spacing seems to matter here? lol